From 4070b0bffea510d86f2117813a866620013b8511 Mon Sep 17 00:00:00 2001
From: Brad Bishop <bradleyb@fuzziesquirrel.com>
Date: Tue, 18 Aug 2020 18:36:01 -0400
Subject: [PATCH] Avoid Power9 CFAM I2C buses

If fru-device is allowed to scan these buses "SCL is stuck low, exit
recovery" messages fill the kernel log and the buses becomes
inaccessible.  Further investigation is required before enabling.

Upstream-Status: Inappropriate
Signed-off-by: Brad Bishop <bradleyb@fuzziesquirrel.com>
---
 blacklist.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/blacklist.json b/blacklist.json
index f112a7b..f9e81a4 100644
--- a/blacklist.json
+++ b/blacklist.json
@@ -1,3 +1,3 @@
 {
-    "buses": []
+    "buses" : [ 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 200, 201, 202, 203, 204, 205, 206, 207, 208, 209, 210, 211, 212, 213, 214 ]
 }
--
2.26.2

